pkg/runner/runner.go

```diff
@@ -161,7 +167,12 @@ func Run(cfg *Config) error {
 	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
 	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
Should we move these lines into the router as well?

Reply:
I would not add them at all to non-debug builds

Reply:
I would put them behind the auth handler, keep them on by default, but add a cli flag to disable

Reply:
I agree with @cevian but I think adding a cli flag for this maybe be overdo.

Reply:
Putting them behind auth handler, leaving out cli flag since this is outside the scope of this PR.